Update de.html 1250-1500 [Waiting for approval]

randomcmd opened the pull request.

randomcmd committed:
Update de.html 1250-1500 [Waiting for approval]

its-nico commented on de.html:
```diff
@@ -1258,7 +1258,7 @@
   Der Konsens in der Spaced Repetition-Community sagt folgendes: <b>Schreibe deine eigenen Karten nach einer Zeit</b>
```
```suggestion
  Der Konsens in der Spaced Repetition-Community ist: <b>Mache irgendwann deine eigenen Lernkarten.</b>
```

herrhundt replied:
```suggestion
  Der Konsens in der Spaced Repetition-Community sagt folgendes: <b>Du solltest nach einiger Zeit deine eigenen Karten erstellen</b>
```

its-nico commented on de.html:
```diff
@@ -1361,13 +1361,13 @@
 	Details über die Leben deiner Freunde
```
```suggestion
	Details über das Leben deiner Freunde
```

its-nico replied:
Klingt besser, auch wenn "die Leben", glaube ich, auch korrekt wäre.

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

its-nico commented on de.html:
```diff
@@ -1398,8 +1398,8 @@
-	Das könnte dir jetzt zu philosophisch sein, deshalb hier
-	spezielles Beispiel für dem <i>Warum</i> hinter einem <i>Was</i>:
+	Das könnte dir jetzt zu philosophisch sein, deshalb hier ein
+	spezielles Beispiel für das <i>Warum</i> hinter einem <i>Was</i>:
```
```suggestion
	ein paar Beispiele für ein <i>Warum</i> hinter einem <i>Was</i>:
```

its-nico replied:
Es sind mehrere Beispiele, außerdem passt der Text so ins Textfeld.

its-nico committed:
Udate de.html L1264-1358/1373-1398/1405-1409/1415-1422/1454-1496

Diese Zeilen konnte ich in Adrians pull request nicht kommentieren, da sie eingeklappt waren, und das dann anscheinend nicht funktioniert. In diesen Zeilen habe ich ein paar Dinge verbessert bzw. andere Formulierungen vorgeschlagen

its-nico commented on de.html:
```diff
@@ -1409,7 +1409,7 @@
 	<b>WAS:</b> Programmierung
 	<br>
-	<b>WARUM:</b> um Geld zu verdienen um dir Essen leisten zu können
+	<b>WARUM:</b> um Geld zu verdienen damit du dir Essen leisten zu kannst
```
```suggestion
	<b>WARUM:</b> um Geld zu verdienen, damit du dir Essen leisten kannst
```

its-nico commented on de.html:
```diff
@@ -1409,7 +1409,7 @@
 <words x=130 y=210 w=350 style="text-align:left" no-bg class="comic_text">
 	<b>WAS:</b> Programmierung
```
```suggestion
	<b>WAS:</b> eine Programmiersprache
```

its-nico replied:
Passt so besser zum oberen Punkt.

its-nico commented on de.html:
```diff
@@ -1469,22 +1468,20 @@
 	<b>TinyCards (App)</b>
 	<br>
-	Pros: schönes Design, einfach zu nutzen
+	Pros: wundervolles Design, einfach zu nutzen
```
Würde "schönes" lassen, oder auch "tolles". Im Original steht "beautiful", das wäre übersetzt aber ja dann eher "wunderschönes"

herrhundt replied:
```suggestion
	Pros: schönes Design, einfach zu nutzen
```

its-nico commented on de.html:
```diff
@@ -1469,22 +1468,20 @@
+	Pros: wundervolles Design, einfach zu nutzen
 	<br>
-	Cons: max 150 Karten pro Deck, lässt dich nicht <i>selbst</i> entscheiden, ob du eine Karte richtig hast
+	Cons: maximal 150 Karten pro Deck, lässt dich nicht <i>selbst</i> entscheiden, ob du eine Karte richtig erraten hast
```
```suggestion
	Cons: maximal 150 Karten pro Deck, lässt dich nicht <i>selbst</i> entscheiden, ob du eine Karte richtig gelernt hast
```

its-nico replied:
Die Inhalte sollen ja erlernt werden, nicht erraten

its-nico commented on de.html:
```diff
@@ -1469,22 +1468,20 @@
 <words x=110 y=620 w=360 h=40 no-bg fontsize=20>
-	* <i>Anti</i>-Haftungsausschluss: Ich bin mit <i>keinem</i> davon geschäfftlich verbunden.
-	ich finde sie nur cool &amp; hilfreich!	
+	* <i>Anti</i>-Haftungsausschluss: Ich bin mit <i>keinen</i> Unternehmen davon geschäfftlich verbunden.
```
```suggestion
	* <i>Anti</i>-Haftungsausschluss: Ich bin mit <i>keinem</i> dieser Unternehmen geschäftlich verbunden.
```

noahdyj committed:
Update de.html L1364

noahdyj committed:
Update de.htlm L1405

noahdyj committed:
Update de.html L1412

its-nico committed:
Merge branch 'randomcmd-patch-2' into randomcmd-patch-3

its-nico commented on de.html:
```diff
@@ -1399,7 +1401,9 @@
 <words x=10 y=10 w=430 h=60>
 	Das könnte dir jetzt zu philosophisch sein, deshalb hier ein
+
 	spezielles Beispiel für ein <i>Warum</i> hinter einem <i>Was</i>:
```
```suggestion
	paar Beispiele für ein <i>Warum</i> hinter einem <i>Was</i>:
```

its-nico committed:
Merge pull request #25 from herrhundt/its-nico-patch-3

Udate de.html L1264-1358/1373-1398/1405-1409/1415-1422/1454-1496

its-nico committed:
Update de.html

Co-authored-by: herrhundt <59929867+herrhundt@users.noreply.github.com>

its-nico commented on de.html:
```diff
@@ -1399,20 +1401,22 @@
 	<b>WAS:</b> Programmierung
 	<br>
-	<b>WARUM:</b> um Geld zu verdienen um dir Essen leisten zu können
+	<b>WARUM:</b> Um Geld zu verdienen damit du dir Essen leisten zu kannst
```
```suggestion
	<b>WARUM:</b> Um Geld zu verdienen, damit du dir Essen leisten kannst
```

its-nico committed:
Update de.html

its-nico committed:
Update de.html

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html
